### DISCUSSION TITLE
How to completely reset values and uninstall

### DISCUSSION THREAD

**Discussion by LilRoosta** [MonitorControl/MonitorControl#1592] [Q&A] [answered]
I wanted to uninstall so the changes i made would reset and go back to normal as i didn't need to use it anymore, but the settings are still applied after uninstalling. I am aware of the "reset settings" button in the displays tab, but that only resets my brightness, not contrast (which is what i really want to reset). Is there any way to thoroughly uninstall while also resetting everything? If I cant fully reset contrast, then at least what is the default value so I can put it back?

**Comment by waydabber** [comment, marked as answer]
Hi there - you should simply remove the app. There is no universal default value for contrast (or brightness), each display has its own default value (typically 50, 70, 75, 100). You should find a reset to factory defaults option in your display's OSD.

As an alternative you can download [BetterDisplay](https://github.com/waydabber/BetterDisplay) which has a DDC factory reset option - this instructs the display to reset all settings (should be the same as the reset using the display's OSD). This works with most displays.

<img width="373" alt="Screenshot 2024-08-18 at 18 04 14" src="https://github.com/user-attachments/assets/df373ecc-01fa-47dc-bb08-5298532bc9de">

**Reply by LilRoosta** [reply]
Thanks for the help. I honestly had no idea that the controls change MY MONITORS settings and not just the image being sent to the monitor, which is why i wanted to reset in the first place. (I guess reading more would have helped me out lol) So this is actually a lot better because i though i was changing some hidden mac brightness/contrast settings.

**Reply by waydabber** [reply]
You are welcome. :)
